docs/tutorials/forecasting/extended_tutorial.md

```diff
@@ -845,11 +845,11 @@ from mxnet.gluon import HybridBlock
 from gluonts.core.component import validated
 from gluonts.dataset.loader import TrainDataLoader
 from gluonts.model.predictor import Predictor
-from gluonts.mx.batchify import as_in_context, batchify
-from gluonts.mx.model.estimator import GluonEstimator
-from gluonts.mx.model.predictor import RepresentableBlockPredictor
-from gluonts.mx.trainer import Trainer
-from gluonts.mx.util import copy_parameters, get_hybrid_forward_input_names
+from gluonts.mx import as_in_context, batchify
+from gluonts.mx import GluonEstimator
+from gluonts.mx import RepresentableBlockPredictor
+from gluonts.mx import Trainer
+from gluonts.mx import copy_parameters, get_hybrid_forward_input_names
```

Comment:
I guess these could be all in one? As in

```
from gluonts.mx import (
   ...
   ...
)
```

Reply:
Concatenated ✔️